_posts/2015-11-24-booting-discovery-over-http.md

```diff
@@ -0,0 +1,87 @@
+---
+layout: post
+title: 
+date: 2015-11-24 11:14:15
+author: Foreman
+tags:
```

Comment:
They're to automatically add credit where a blog was originally posted somewhere else first - see the guest* metadata in https://github.com/theforeman/theforeman.org/blob/gh-pages/_posts/2015-06-11-coreos-cluster-deployments-with-foreman.md for an example. However, I think this is an entirely new post, right?